Header.h

```diff
@@ -0,0 +1,146 @@
+#pragma once
+#include <iostream>
+#include <string>
+using namespace std;
+class Movie
+{
+public:
+    int id;
+    string title;
+    string ranking;
+    int release_date;
+    int number_of_tickets;
+    int ticket_price_in_uan;
+    string comment;
+    int GetID(){
+        return id;
+    }
+    string GetTITLE(){
+        return title;
+    }
+    string GetRANKING(){
+        return ranking;
+    }
+    int GetRELEASE_DATE(){
+        return release_date;
+    }
+    int GetNUMBER_OF_TICKETS(){
+        return number_of_tickets;
+    }
+    int GetTICKET_PRICE_IN_UAN(){
+        return ticket_price_in_uan;
+    }
+    string GetCOMMENT(){
+        return comment;
+    };
+    Movie() {};
+    enum filmType
+    {
+        ACTION,
+        ADVENTURE,
+        COMEDY,
+        CRIME,
+        CRIMINAL,
+        DETECTIVE,
+        DRAMA,
+        FAMILY,
+        FANTASY,
+        HORRORS,
+        MELODRAMA,
+        ROMANCE,
+        THRILLER
+    };
+    void calculateProfit()
+    {
+        int a = number_of_tickets;
+        int b = ticket_price_in_uan;
```

Comment:
A variable must not consist of 1 character. Variable name should clearly describes its purpose